hadoop-hdds/common/src/main/resources/ozone-default.xml

```diff
@@ -2247,11 +2247,29 @@
     <name>hdds.x509.ca.rotation.time-of-day</name>
     <value>02:00:00</value>
     <tag>OZONE, HDDS, SECURITY</tag>
-    <description>Time of day to start the rotation. Default 02:00 AM to avoid impacting
-      daily workload. The supported format is 'hh:mm:ss', representing hour, minute,
+    <description>Time of day to start the rotation. Default 02:00 AM to avoid
+      impacting
+      daily workload. The supported format is 'hh:mm:ss', representing hour,
+      minute,
       and second.
     </description>
   </property>
+  <property>
+    <name>ozone.scm.security.service.address</name>
```

Comment:
`ozone.scm.security.service.address` This is duplicate config, already there is one. Don't see used in the new code, may be by mistake added. Please remove it.